doctr/utils/data.py

```diff
@@ -84,8 +89,14 @@ def download_from_url(
         logging.info(f"Using downloaded & verified file: {file_path}")
         return file_path
 
-    # Create folder hierarchy
-    folder_path.mkdir(parents=True, exist_ok=True)
+    try:
```

Comment:
It would be great if you add an test case for `download_from_url` in tests/common/test_utils_data.py (you need to create this file) :+1: 

you can use this url as test download : 
https://github.com/mindee/doctr/releases/download/v0.5.1/word-crop.png

Reply:
If what we are trying to do here is to check whether the folder is writable, I think we shouldn't do a `try except`. `os.access` provides information about a folder writability for instance. If we're trying to cover other things, perhaps the OSError catching makes sense, but it would be better to avoid that!

And yes, we need a unittest for this :sweat_smile:

Reply:
I decided to stick with catching OSError, because `os.access` doesn't help when directory does not exist